release-notes/2023-03/README.md

```diff
@@ -0,0 +1,214 @@
+# SystemLink Enterprise 2023-03 Release Notes
+
+The 2023-03 bundle for SystemLink Enterprise has been published to <https://niedge01.jfrog.io>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## New Features and Behavior changes
+
+- Support for Kubernetes 1.23
+
+- Support for both PostgreSQL 13 and 14.
+
+- Users can now export data tables to CSV
+
+- Product, results, and steps tables can be queried using the table query builder.
+
+- OpenTelemetry metrics are now available for the following services
+    - testmonitor
+    - dataframeservice
+    - Kafka
+
+- The endpoint to view claims for the current logged in user, `/oidc/userinfo`, has been removed and replaced with `oidc/user-info`.
+
+- Kafka UI is now available for debugging and monitoring
+
+- Kafka S3 sink connectors are automatically deleted if a data table is marked `endOfData`.
+
+- Deleted Kafka S3 sink connectors are automatically reestablished if connectors are manually deleted.
+
+- The default memory provided to Kafka topic operator has increased from 256MB to 512MB to increase the total number of tables that can be written to by the Data Frame Service.
+
+## Helm Chart Breaking Changes
+
+- `systemlink 0.11.81`
```

Comment:
This has been rebuilt.

```suggestion
- `systemlink 0.11.85`
```

The only change in this release bundle is a new version of the dataframeservice chart (0.8.123), which pulled in a new version of the kafka-ui chart (0.5.4).

https://dev.azure.com/ni/DevCentral/_build/results?buildId=4433740&view=logs&j=c139e765-cd1b-5f1c-f29e-7c4d3e9388b9&t=eca95393-96ed-5496-e1f5-f82b94ffef11